packages/typespec-ts/test/modularIntegration/generated/headers/repeatability/generated/src/models/index.ts

```diff
@@ -1,4 +1,5 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+export * from "./models.js";
```

Comment:
Why did we start generating this emtpy models.ts file?

Reply:
should be removed now